system/keyboardlayouts/chinese.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!--
+Please use English language names instead.
+Default font lacks support for all characters
+-->
+<keyboardlayouts>
+  <layout language="Chinese" layout="BasePY" codingtable="BasePY">
```

Comment:
Ah sorry I just realized that the `layout` attribute is not an addition of this PR but exists in all keyboardlayouts. Nevermind my comment then.